libs/openFrameworks/3d/ofEasyCam.cpp

```diff
@@ -266,49 +269,65 @@ bool ofEasyCam::getInertiaEnabled() const{
 //----------------------------------------
 void ofEasyCam::updateTranslation(){
 	if(bApplyInertia){
-		moveX *= drag;
-		moveY *= drag;
-		moveZ *= drag;
-
-		if(ABS(moveZ) >= minDifference){
-			bIsBeingScrolled = true;
-		} else {
-			bIsBeingScrolled = false;
-		}
-
-		if(ABS(moveX) <= minDifference && ABS(moveY) <= minDifference && ABS(moveZ) <= minDifference){
+		translate *= drag;
+//		bDoZoom = (ABS(translate.z) >= minDifference);
+		if(ABS(translate.x) <= minDifference && ABS(translate.y) <= minDifference && ABS(translate.z) <= minDifference){
```

Comment:
Can I request using `std::abs`, rather than the `ABS` macro, as we're working on deprecating `ABS` etc.

See #5624 for conflicting changes.